src/helm/benchmark/data_overlap/light_scenario.py

```diff
@@ -0,0 +1,56 @@
+from dataclasses import dataclass
+from typing import List, Optional
+
+try:
+    from scenarios.scenario import ScenarioSpec
+except Exception:
+    from helm.benchmark.scenarios.scenario import ScenarioSpec
+
+
+@dataclass(frozen=True)
+class LightInstance:
+    """
+    A lighter `Instance` with only text fields.
+    """
+
+    """The input"""
```

Comment:
Tip: docstrings go _below_ e.g.

```
input: str
"""The input"""
```

This is a non-PEP convention that is supported by VSCode, mkdocs, etc. ([reference](https://stackoverflow.com/a/69754746)

Reply:
Thanks for the note, updated